conda.recipe/meta.yaml

```diff
@@ -54,15 +58,15 @@ test:
     - mock
     - pandas
     - pytest >=2.9.0
-    - pytest-cov ==1.8.1
-    - pytest-selenium >=1.2.1
+    - pytest-cov ==1.8.1       [unix]
```

Comment:
Yes, otherwise it wont build. The recipe does not actually run tests, so these dependencies are currently not necessary. So ok for now. If we do make the recipe run tests we should probably add a pytest-conv windows package.